Rome-Blockchain-Labs/terminal-widgets
# RT-162 romebridge integration

- use romebridge sdk for parent-widget communication

To test:
1. `yarn && yarn dev`
2.  checkout `https://github.com/Rome-Blockchain-Labs/rome-frontend/tree/RT-83-sherpa-button-bridge-logic`
3. switch https://github.com/Rome-Blockchain-Labs/rome-frontend/blob/1691f2b2419898ab9d67fa3147ecdfd0802003cc/src/widgets/SherpaCash/index.tsx#L35  to use `http://localhost:3001/'
4. `yarn start`
5. Open shepa widget and test clicking buttons
6. Should switch b/n compliance and home screen
